common/sgx/edl/thread.edl

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** sgx/thread.edl:
+**
+**     Internal OCALLs to be used by liboehost/liboecore for thread operations.
+**
+**     Note that this functionality is not inherently specific to SGX, it is
```

Comment:
Should the EDL file be in a non-SGX-specific directory then?  (Analogy is that we have other header files with APIs that currently return "not implemented" on optee, this seems like it should be in the same category)
If this is punted to a future PR, then I'd recommend filing a GitHub issue, and optionally referencing that GitHub issue # in the comments in the edl.